### PR TITLE
change README to tell about coverage only, not full project config

### PR DESCRIPTION
**Pull request by geoffray-adde-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= C++ example project scanned on SonarCloud with code coverage using Azure Pipelines
+= C++ example project for generating test coverage with vscoverage on Azure Pipelines
```
Comment (arseniy-sonar) [inline README.adoc:1]:
```suggestion
= C++ example project for generating test coverage with VisualStudio coverage tool on Azure Pipelines
```
IMHO vscoverage isn't a thing, and we can afford to spell the full name of the concept

File: README.adoc [hunk 2]
```diff
@@ -50,15 +30,6 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
 - https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]
 
-= Windows\MSBuild
```
Comment (arseniy-sonar) [inline README.adoc:53]: Why remove this section?

File: README.adoc [hunk 3]
```diff
@@ -8,36 +8,16 @@
...
-.. Add a task to download the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
-.. Wrap your compilation with the Build Wrapper
-.. Add a `VSTest` task to launch your tests. Be sure to enable code coverage and to convert the result file into an XML file
-.. Add the `SonarCloudAnalyze` task
+*The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_windows-vscoverage-msbuild-azure-sc[SonarCloud]!*
+
+NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Some analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=topic%3Aconfiguration+msbuild-azure&type=all&language=&sort=[configuration examples] are available. +
+Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.
+
+To get coverage and feed it to the sonar-scanner, use the following procedure.
```
Comment (arseniy-sonar) [inline README.adoc:16]:
```suggestion
To get coverage and feed it to the Sonar Scanner, use the following procedure.
```

File: README.adoc [hunk 4]
```diff
@@ -8,36 +8,16 @@
...
+
+. In your azure-pipelines.yml file, Initialize the extra property `sonar.cfamily.vscoveragexml.reportsPath` with the path that will contain your XML coverage report
```
Comment (arseniy-sonar) [inline README.adoc:18]:
```suggestion
. In your azure-pipelines.yml file, initialize the extra property `sonar.cfamily.vscoveragexml.reportsPath` with the path that will contain your XML coverage report
```

File: README.adoc [hunk 5]
```diff
@@ -8,36 +8,16 @@
...
+. Add a `VSTest` task to launch your tests. Be sure to enable code coverage and to convert the result file into an XML file
+. Add the `SonarCloudAnalyze` task
```
Comment (arseniy-sonar) [inline README.adoc:20]:
```suggestion
. Make sure the `SonarCloudAnalyze` task runs after the `VSTest` task
```
If we assume the analysis is configured, `SonarCloudAnalyze` task is already there.

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (geoffray-adde-sonarsource) [reply README.adoc:53]: as agreed, closing this

Comment (arseniy-sonar) [reply README.adoc:53]: After f2f discussion. It is no longer relevant.

**Pull request merged**